nemo/collections/nlp/parts/megatron_trainer_builder.py

```diff
@@ -95,10 +95,10 @@ def _plugins(self) -> list:
 
         return plugins
 
-    def create_trainer(self) -> Trainer:
+    def create_trainer(self, callbacks=(CustomProgressBar(),)) -> Trainer:
```

Comment:
Don't instantiate objects in function signature. Default to None, check if None inside the function and the. Instantiate the value you need. Only primitive values are safe in function signature. https://leimao.github.io/blog/Python-Default-Argument-Mutable-Object/

Reply:
Fixed